src/amo/components/Routes/index.js

```diff
@@ -147,6 +147,12 @@ const Routes = ({ _config = config }: Props = {}) => (
       component={CategoriesPage}
     />
 
+    <Route
+      exact
+      path="/:lang/:application/category/"
+      component={(props) => <SearchPage {...props} isForCategory />}
```

Comment:
It'd be best to create a new page component (`CategoryPage`?). We'll want to use `HeadLinks` and maybe `HeadMetaTags` in the future.

Reply:
Again, I'm not sure if we need these if the only point is to have a page for the crawler to crawl. Will the page be penalized if it doesn't contain those?

Reply:
FWIW, I did start with a new page component and then realized "hey, we don't need that", but I guess maybe we do.

Reply:
Yeah, I think we do need a new page component. We need `HeadLinks` to tell the crawlers what to crawl.